features/26-mfa/api.md

```diff
@@ -37,6 +37,16 @@ mfa:
   #
   # Default is 'off'
   enforcement: 'off'
+  totp:
+    # The maximum number of TOTP authenticator the user can register.
+    # Valid values is [0, 10].
+    # Default is 1.
+    maximum: 1
```

Comment:
Do we need to provide a default value on maximum number of authenticators? If the user just opt-in with enforcement = optional/required, the default settings would require user to implement TOTP/OOB flow UI.

Reply:
Introduce `maximum` to limit the number of authenticators. The default value is 1. Cap at 20. Change the default value of `totp.maximum` and `oob.maximum` to 0 so the developer must choose which authenticator they want to enable as enabling the authenticator requires the developer to implement a UI flow. Effectively, the developer must opt-in the authenticator they plan to support.

## At most 1 TOTP or 1 OOB

```yaml
mfa:
  enforcement: "optional"
  maximum: 1
  totp:
    maximum: 1
  oob:
    maximum: 1
```

## At most 1 TOTP and 1 OOB

```yaml
mfa:
  enforcement: "optional"
  maximum: 2
  totp:
    maximum: 1
  oob:
    maximum: 1
```

## At most 1 TOTP

```yaml
mfa:
  enforcement: "optional"
  totp:
    maximum: 1
```

## At most 1 OOB

```yaml
mfa:
  enforcement: "optional"
  oob:
    maximum: 1
```